suite-native/video-assets/package.json

```diff
@@ -4,10 +4,10 @@
     "private": true,
     "license": "See LICENSE.md in repo root",
     "sideEffects": false,
+    "type": "module",
```

Comment:
This probably shouldn't be here

Reply:
![Screenshot 2023-02-16 at 11 48 57](https://user-images.githubusercontent.com/26143964/219344466-f3d11a2b-2b74-412d-936a-11db286305c2.png)

So `"type": "module"` should be only in packages with `module-` prefix? Then if I'm not mistaken, it's wrongly used in `suite-native/navigation`. I remove it there then, right?

Reply:
It shouldn't be anywhere, treating whole packages as es modules leads to more issue than benefits. It has nothing to do with packages with `module-` prefix. In suite-native it's probably fine because Metro will handle this just fine...